rosdep/python.yaml

```diff
@@ -9322,7 +9325,11 @@ python3-sense-hat-pip:
     pip:
       packages: [sense-hat]
 python3-serial:
-  alpine: [py3-serial]
+  alpine:
+    '*': [py3-pyserial]
+    '3.11': [py3-serial]
+    '3.14': [py3-serial]
+    '3.8': [py3-serial]
```

Comment:
[optional] Maybe this could be 
```
'*': [py3-serial]
'3.17': [py3-pyserial]
```

Reply:
As future Alpine versions will also have `py3-pyserial` in official repository, explicitly specifying old versions would be better.